98Installation/sec00virtualbox.md

```diff
@@ -32,25 +33,42 @@ Then you can start your VM for the first time it will ask you to specify the Ubu
 
 When installing Ubuntu select the `Minimal installation` and check the box which says `Download updates while installing Ubuntu`. Select `Erase disk and install Ubuntu` and `Encrypt the new Ubuntu installation` boxes but make sure you write down you security key password as if you lose this you will lose access to the VM.
 
-Once complete you'll need to rshutdown and restart the VM.
+Once complete you'll need to shutdown and restart the VM.
 
 ### Installing required packages
 
-Then you need to install the following packages using the apt-get:
-```
-sudo apt-get update
-sudo apt-get install build-essential
-sudo apt-get install cmake
-sudo apt-get install git
+Then you need to install `build-essential`, `cmake` and `git` using apt-get. In a terminal run the following commands:
+
+``` bash
+sudo apt-get update 
+sudo apt-get install build-essential cmake git
 ```
 
 ### Misc
 
+#### Enabling copy and paste between host and virtual machine
+
 To enable copy and paste between host machine (your laptop) and the VM you may need to complete the following steps:
 Power off the VM, then Settings -> Advanced -> and set Shared Clipboard to Bidirectional
 Restart the VM and see if you can copy and paste (in Ubuntu paste is `Shift-Ctrl-C` and `Shift-Ctrl-V`).
 If you are unable to copy to from your host the the VM then the following may need to be done:
-```
+
+``` bash
 sudo apt-get install virtualbox-guest-x11-hwe
 sudo VBoxClient --clipboard
 ```
+
+#### Workaround for small font and window size
+
+If the fonts and window size on the virtual machine are unreadabe because they are too small or too large first try installing the above `virtualbox-guest-x11-hwe` and then restaring the virtual machine. If this doesn't resolve the issue and the font size is still unusable then the following workaround may help.
+
+Run the following in a terminal:
+
+``` bash
+sudo apt-get install gnome-tweaks
+```
+
+This will install a small a small GUI that can be opened using the desktop search for `Tweaks`. Then under `Fonts` there is a scaling factor that can be adjusted as needed. 
```

Comment:
```suggestion
This will install a small GUI that can be opened using the desktop search for `Tweaks`. Then under `Fonts` there is a scaling factor that can be adjusted as needed. 
```